test/PspHgh.jl

```diff
@@ -91,18 +96,24 @@ end
     @test (res[1] < res[2]) == (res[3] < res[2])
 
     for i in 1:2, l in 0:2
-        qcut = qcut_psp_projector(psp, i, l)
-        res = eval_psp_projector_fourier.(psp, i, l, [qcut - ε, qcut, qcut + ε])
-        @test (res[1] < res[2]) == (res[3] < res[2])
+        qcut1 = qcut_psp_projector(psp, i, l)
+        res1 = eval_psp_projector_fourier.(psp, i, l, [qcut1 - ε, qcut1, qcut1 + ε])
+        @test (res1[1] < res1[2]) == (res1[3] < res1[2])
     end
 end
 
-@testset "Agreement of polynomial implementation and eval functions" begin
-    psp = load_psp("hgh/lda/Si-q4")
-    Qloc = psp_local_polynomial(Float64, psp)
-    evalQloc(q) = let t = q * psp.rloc; Qloc(t) * exp(-t^2 / 2) / t^2; end
+@testitem "Agreement of polynomial implementation and eval functions" tags=[:psp] begin
+    using LinearAlgebra
+    using DFTK: load_psp, eval_psp_projector_fourier, eval_psp_local_fourier
+    using DFTK: eval_psp_projector_real, psp_local_polynomial, eval_psp_local_real
+    using DFTK: psp_projector_polynomial, qcut_psp_projector, qcut_psp_local
+    using DFTK: eval_psp_energy_correction, count_n_proj_radial
+
+    psp0 = load_psp("hgh/lda/Si-q4")
```

Comment:
Why the rename ?

Reply:
Strange, I thought the whole point of this testrunner busyness was to avoid putting things in global scope. What does it clash with ? Maybe we should rename that instead ?

Reply:
Yeah as I mentioned elsewhere. My strategy would be to wrap the `psp` in the code directly above the loop in a `let` block.